src/code/audio/code_8019AF00.c

```diff
@@ -1453,15 +1453,15 @@ u8 sPlayedOcarinaSongIndexPlusOne = 0;
 u8 sMusicStaffNumNotesPerTest = 0;
 u8 D_801D8530 = false;
 u32 D_801D8534 = 0;
-u8 sIsOcarinaNoteChanged = false;
+u8 sOcarinaDropInputTimer = false;
 
 OcarinaNote sScarecrowsLongSongNotes[108] = {
     { OCARINA_PITCH_NONE, 0, 0, 0, 0, 0 },
     { OCARINA_PITCH_NONE, 0, 0, 0, 0, 0 },
 };
 
 OcarinaNote* gScarecrowLongSongPtr = sScarecrowsLongSongNotes;
-u8* gScarecrowSpawnSongPtr = (u8*)&sOcarinaSongNotes[OCARINA_SONG_SCARECROW];
+u8* gScarecrowSpawnSongPtr = (u8*)&sOcarinaSongNotes[OCARINA_SONG_SCARECROW_SPAWN];
```

Comment:
Hmm does `gScarecrowSpawnSongPtr` have to be a `u8*`? Can it be an `OcarinaNote*`?

Reply:
This being a `u8` comes more into play with `sram` and `z_message`. For consistency, I'm going to rename `scarecrowsSong` -> `scarecrowSpawnSong`, but there are a few for-loops that indicates that this is actually a `u8`, see:
https://github.com/zeldaret/oot/blob/66946534a14334fc1c1e16ace252de7d167b2420/src/code/z_message_PAL.c#L2698-L2702
https://github.com/zeldaret/oot/blob/66946534a14334fc1c1e16ace252de7d167b2420/src/code/z_sram.c#L426-L431
and:
https://github.com/zeldaret/mm/blob/45eed680d613642bc3e1591f95902f6d57e0f8e6/include/z64save.h#L169

Reply:
Curious. Seems strange to me, but if that is what's needed :shrug: